src/glog/logging.h.in

```diff
@@ -360,6 +360,9 @@ DECLARE_int32(minloglevel);
 // default logging directory.
 DECLARE_string(log_dir);
 
+// Allow writing to the same logfile, using date-only broad suffixes.
+DECLARE_bool(usedatefilenames);
```

Comment:
Somehow, I felt like this flag should have more underscores in it... Then I looked higher in the file, and saw a bunch that look like it, so I guess that's fine. :smiley:

Maybe @shinh or @ukai have a style that they prefer and are trying to go toward (existing flags are hard to change, to avoid breaking existing user scripts and such)?